internal/sql/repository/pipelineConfig/CdWorfkflowRepository.go

```diff
@@ -357,7 +357,15 @@ func (impl *CdWorkflowRepositoryImpl) FindLatestCdWorkflowByPipelineIdV2(pipelin
 	// TODO - Group By Environment And Pipeline will get latest pipeline from top
 	return cdWorkflow, err
 }
-
+func (impl *CdWorkflowRepositoryImpl) FindAllTriggeredWorkflowInLast24Hour() (cdWorkflowCount int, err error) {
+	var workflowCount int
+	query := "SELECT count(DISTINCT(cd_workflow_id)) FROM cd_workflow_runner WHERE started_on > now() - interval '1 day'"
```

Comment:
cd_workflow_runner has 3 runner types: PRE/ POST/ DEPLOY. we'll fetch the deployments only I guess. verify it from Product.